src/main/java/org/example/GameStoreConsole.java

```diff
@@ -0,0 +1,123 @@
+package org.example;
+
+import org.example.entity.Game;
+import org.example.repository.dao.GameRepository;
+import org.example.services.GameService;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class GameStoreConsole {
+    private GameService gameService = new GameService(new GameRepository());
+    private Scanner scanner = new Scanner(System.in);
+
+    public void start() {
+        while (true) {
+            System.out.println(
+                    """
+                            Виберіть дію:
+                            1. Додати гру
+                            2. Видалити гру
+                            3. Пошук гри за назвою
+                            4. Фільтрувати ігри за ціною
+                            5. Показати всі ігри, відсортовані за датою додавання
+                            6. Перегляд списку всіх доступних ігор
+                            7. Вийти"""
+            );
+
+            int choice = scanner.nextInt();
+            scanner.nextLine();
+
+            switch (choice) {
+                case 1:
+                    addGame();
+                    break;
+                case 2:
+                    deleteGame();
+                    break;
+                case 3:
+                    searchGameByName();
+                    break;
+                case 4:
+                    filterGamesByPrice();
+                    break;
+                case 5:
+                    listAllGamesSortedByCreationDate();
+                    break;
+                case 6:
+                    listAllGames();
+                    break;
+                case 7:
+                    System.exit(0);
+            }
+        }
+    }
+
+    public void addGame() {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
you already have class level scanner var,  what is the point of creating a new one here